### ISSUE TITLE
Add support for KingdomsX Champion

### ISSUE THREAD

**Issue by LRoffic** [Nathat23/StackMob-5#172]
Hello,
I would know if it's possible to add compatibility with the mob champion of [KingdomsX plugin](https://www.spigotmc.org/resources/kingdomsx.77670/updates)
Thank in advance, cordially LRoffic

**Comment by Nathat23** [comment]
Doesn't seem to have any kind of API/maven repo? You'd have to ask the developer of that plugin.

**Comment by LRoffic** [comment]
Yes it have an API: https://github.com/CryptoMorin/KingdomsX/wiki/API
